rag_experiment_accelerator/config/config.py

```diff
@@ -145,6 +147,8 @@ def __init__(
         for key, value in config_json.items():
             logger.debug(f"Configuration setting: {key} = {value}")
 
+        self.QUERY_EXPANSION = config_json.get("query_expansion", "Disabled")
```

Comment:
Is the default `Disabled` or `disabled`?
You've used both in this code and config. I suspect it doesn't matter (and that any value other than the other special values) is the same as disabled. However, probably good to be consistent.

Reply:
lower case is better. thanks for noticing that.